core/src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -120,7 +120,7 @@ class CassandraReadJournal(system: ExtendedActorSystem, cfg: Config, cfgPath: St
   private val serialization = SerializationExtension(system)
   implicit private val ec =
     system.dispatchers.lookup(querySettings.pluginDispatcher)
-  implicit private val materializer = ActorMaterializer()(system)
+  implicit private val sys: ActorSystem = system
```

Comment:
It's somewhat wrong that we didn't use the pluginDispatcher as the dispatcher for the Materializer previously.
To honor that setting I think it's actually better to create a separate materializer here.

Reply:
For now akka-stream provides two factories for materializers:
- `def createMaterializer(systemProvider: ClassicActorSystemProvider): Materializer`
- `def createMaterializer(contextProvider: ClassicActorContextProvider): Materializer`
so I don't know how to create the materializer with above dispatcher.
Could you explain me how to do it?

Reply:
Sorry, I was wrong. Defining a dispatcher doesn't require a separate materializer. It's an attribute on the stream (Source or RunnableGraph). I don't know if we already set that here, otherwise we should. Then the implicit ActorSystem can be used.

```
.withAttributes(ActorAttributes.dispatcher(querySettings.pluginDispatcher)
```

Reply:
It seems that the attribute is set.

Reply:
good, sorry for the noise then